### DISCUSSION TITLE
Calendrier ICS

### DISCUSSION THREAD

**Discussion by Larsene** [JeromeDevome/GRR#373] [General]
Hello,

Nous utilisons GRR depuis un moment, et nous allons devoir mettre en place dans notre bâtiment une régulation du chauffage selon l'occupation. Je souhaitais savoir s'il y a un moyen d'exporter régulièrement le calendrier d'occupation au format ICS pour notre jeedom ?

Merci pour vos réponses et vos pistes de recherche ! :)

**Comment by JeromeDevome** [comment]
Bonjour, en version cloud nous développons ce module.

**Comment by Larsene** [comment]
Merci ! Nous avons développé ce qu'il nous faut ! :-)

Nous allons rajouter également la gestion de portes avec des serrures connectées (pour créer automatiquement un code valide pour les déverrouiller le temps de l'occupation).
